test/commands/app/add/extension.test.js

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const fs = require('fs-extra')
+
+const TheCommand = require('../../../../src/commands/app/add/extension')
+const BaseCommand = require('../../../../src/BaseCommand')
+const generators = require('@adobe/generator-aio-app')
+const { implPromptChoices: availableImplementations } = require('../../../../src/lib/defaults')
+const dataMocks = require('../../../data-mocks/config-loader')
+
+const config = require('@adobe/aio-lib-core-config')
+jest.mock('@adobe/aio-lib-core-config')
+
+jest.mock('fs-extra')
+
+jest.mock('../../../../src/lib/app-helper.js')
+const helpers = require('../../../../src/lib/app-helper.js')
+
+jest.mock('yeoman-environment')
+const yeoman = require('yeoman-environment')
+
+const mockInstantiate = jest.fn()
+const mockRunGenerator = jest.fn()
+yeoman.createEnv.mockReturnValue({
+  instantiate: mockInstantiate,
+  runGenerator: mockRunGenerator
+})
+
+const createFullConfig = (aioConfig = {}, appFixtureName = 'legacy-app') => {
+  const appConfig = dataMocks(appFixtureName, aioConfig)
+  return appConfig
+}
+const createAppConfig = (aioConfig = {}, appFixtureName = 'legacy-app') => {
```

Comment:
`createAppConfig` and `createFullConfig ` function above seem to be doing similar task. Can be combined as a single function with relevant params?